docs/quick-start/configure-mongodb.txt

```diff
@@ -0,0 +1,59 @@
+.. _laravel-quick-start-connect-to-mongodb:
+
+=================================
+Configure Your MongoDB Connection
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Set the Connection String in the Database Configuration
```

Comment:
I: per the [style guide](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#capitalization), I think these steps should use sentence-style capitalization

Reply:
Applies to the other steps/files too